toolchain/driver/format_subcommand.h

```diff
@@ -0,0 +1,39 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_TOOLCHAIN_DRIVER_FORMAT_SUBCOMMAND_H_
+#define CARBON_TOOLCHAIN_DRIVER_FORMAT_SUBCOMMAND_H_
+
+#include "common/command_line.h"
+#include "toolchain/driver/driver_env.h"
+#include "toolchain/driver/driver_subcommand.h"
+
+namespace Carbon {
+
+// Options for the format subcommand.
+//
+// See the implementation of `Build` for documentation on members.
```

Comment:
Maybe a TODO here or on the `Build` command for adding line-range flags and formatting support?

Reply:
I'm putting this on the `Format` call... later PRs are putting a few TODOs there already, so it seems like a good central spot to find missing features.